nvflare/fuel/utils/gpu_utils.py

```diff
@@ -28,8 +33,6 @@ def use_nvidia_smi(query: str, report_format: str = "csv"):
             raise Exception(f"Failed to call nvidia-smi with query {query}", result.stderr)
         else:
             return result.stdout.splitlines()
-    except FileNotFoundError as e:
-        print(f"Failed to call nvidia-smi: {e}")
     return None
 
 
```

Comment:
_parse_gpu_mem also print this "failed" msg we can either remove in this Pr or the later one